app/webpacker/lib/utils/registrationAdmin.js

```diff
@@ -13,6 +13,50 @@ export const APPROVED_ICON = 'check';
 export const CANCELLED_ICON = 'trash';
 export const REJECTED_ICON = 'x';
 
+export const getStatusColor = (key) => {
+  switch (key) {
+    case 'pending':
+      return PENDING_COLOR;
+
+    case 'waiting':
+      return WAITLIST_COLOR;
+
+    case 'accepted':
+      return APPROVED_COLOR;
+
+    case 'cancelled':
+      return CANCELLED_COLOR;
+
+    case 'rejected':
+      return REJECTED_COLOR;
+
+    default:
+      return undefined;
+  }
+};
+
+export const getStatusIcon = (key) => {
+  switch (key) {
+    case 'pending':
+      return PENDING_ICON;
+
+    case 'waiting':
+      return WAITLIST_ICON;
+
+    case 'accepted':
+      return APPROVED_ICON;
+
+    case 'cancelled':
+      return CANCELLED_ICON;
+
+    case 'rejected':
+      return REJECTED_ICON;
+
+    default:
+      return undefined;
+  }
+};
```

Comment:
Just curious: Could these helper methods be useful in other parts of the frontend code as well?

Reply:
Yes, definitely.
I was hesitant to do too many changes here since I have several other pending PRs. But at some point I'll re-use these.